benefits/core/models.py

```diff
@@ -164,7 +108,6 @@ class EnrollmentFlow(models.Model):
         help_text="Primary internal system name for this EnrollmentFlow instance, e.g. in analytics and Eligibility API requests."  # noqa: 501
```

Comment:
Yep, I renamed it to `system_name` since I felt it was a good balance of being descriptive (simply a name) but also that it's for the system to use. There's an extra migration operation to take care of the rename.